i18n/states/TZ.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * TZ states
```

Comment:
Could you please add the full country name here instead of the country code? This is a minor detail but it makes it easier to identify the country and all the other states files follow this pattern.

Reply:
Thanks, done.